src/makeInfinityAdder.js

```diff
@@ -4,7 +4,22 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+    let sum = 0;
+
+  function adder(num) {
+    if (num === undefined) {
```

Comment:
The check for `undefined` is correct as per the task requirements. However, it's worth noting that this check will also consider a call with an explicit `undefined` as a signal to return the sum. This is acceptable given the task description.